pkg/apis/core/types_cloudprofile.go

```diff
@@ -150,3 +155,31 @@ const (
 	// VolumeClassPremium is a constant for the premium volume class.
 	VolumeClassPremium string = "premium"
 )
+
+// VersionClassification is the logical state of a version according to https://github.wdf.sap.corp/kubernetes/kube-docs/wiki/Versioning-Policy
```

Comment:
```suggestion
// VersionClassification is the logical state of a version according to https://github.com/gardener/gardener/blob/master/docs/operations/versioning_policy.md
```
Please remove all referenced to the internal wiki

Reply:
Good catch, thank you